pkg/operator/ceph/file/mds/livenessprobe.sh

```diff
@@ -7,6 +7,7 @@ FILESYSTEM_NAME="{{ .FilesystemName }}"
 KEYRING="{{ .Keyring }}"
 CMD_TIMEOUT="{{ .CmdTimeout }}"
 
+export CEPH_ARGS="--keyring $KEYRING"
 outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --keyring "$KEYRING" --connect-timeout="$CMD_TIMEOUT" --format json)"
```

Comment:
```suggestion
outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --connect-timeout="$CMD_TIMEOUT" --format json)"
```

if you keep the `CEPH_ARGS` change

Reply:
It worked with `--keyring "$KEYRING"` in the cli, but will remove it and add a comment